lib/web_ui/lib/src/engine/view_embedder/global_html_attributes.dart

```diff
@@ -39,6 +39,9 @@ class GlobalHtmlAttributes {
     //     document.querySelector('flutter-view[flt-view-id="$viewId"]')
     rootElement.setAttribute(flutterViewIdAttributeName, viewId);
 
+    // Allow the flutter view to be focusable;
+    rootElement.setAttribute('tabindex', 0);
```

Comment:
Great call @ditman. Added both :)!

Reply:
It must be always zero to be added in the "natural order" of the browser. Any other value > 0 would break this behavior and grab the focus at the wrong time

Reply:
Suggestions applied!